core/src/main/scala/org/apache/spark/rpc/RpcEnv.scala

```diff
@@ -18,8 +18,11 @@
 package org.apache.spark.rpc
 
 import java.net.URI
+import java.util.concurrent.TimeoutException
 
-import scala.concurrent.{Await, Future}
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.duration._
+import scala.concurrent.{Awaitable, Await, Future}
```

Comment:
nit: you don't need to import `FiniteDuration` explicitly since you're importing `duration._`.  Also you are supposed to order direct class imports before package imports (not just alphabetically), so it should be:

```
import scala.concurrent.{Awaitable, Await, Future}
import scala.concurrent.duration._
```

the intellij import organizer will get this wrong, but aaron davidson wrote a plugin which does it right -- there are instructions for using it here: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports